app/models/TaxCalculationState.scala

```diff
@@ -85,8 +87,11 @@ class TaxCalculationStateFactory @Inject()(val addressControllerConfiguration: A
     }
   }
 
-  def getSaDeadlineStatus(dueDate: DateTime): Option[SaDeadlineStatus] = {
+  def p302DateToCardDateFormat(p302UnixDate: String): String = {
```

Comment:
Can we put this in DateTimeTools and name it more generacally

Reply:
Done